frontend/src/components/Login.tsx

```diff
@@ -13,11 +13,16 @@ const tailLayout = {
 const Login = () => {
     const onFinish = async (values: any) => {
         try {
-            if(!values.confirmed) {
-                console.log('Success:', values);
-                const users = await axios.get("http://localhost:3000/users", values)
-                console.log(users)
+            const user = await axios.get(`http://localhost:3000/users`, {
```

Comment:
Why do I need the post request?

Reply:
As per the specification of the REST, GET is only used to get resources from the server - like querying for boards for a user.

In case of login you send email and password, do comparisons and check on the server to check their validity and then you should generated a new token. This is necessary because tokens should have a TTL(time to live).

Once you generate this token you set it to user and return that token, not the old one. So you are updating user's token on every login request, thus it cannot be GET.

There are also some security issues regarding sending username and password encoded in URL as it happens during GET requests.

Reply:
So, I won't be needing 3 get methods.